### DISCUSSION TITLE
mod not working at all

### DISCUSSION THREAD

**Discussion by Bubbi123543646** [Okabintaro/SubmersedVR#56] [Q&A]
i installed the mod and launched the game but nothing was new it was the same as if i didnt have the mod is it because i have subnautica on epic games???

**Comment by Okabintaro** [comment]
If you are playing on quest with airlink you will have to enable the launch option too. See https://github.com/Okabintaro/SubmersedVR/issues/42#issuecomment-1581106092

**Comment by tumja001** [comment]
Hi, I'm having no luck getting this working either. I've tried installing manually and with Vortex, and I also have the launch option set in steam (-vrmode openvr). The controller joysticks just don't seem to work at all.

**Comment by tumja001** [comment]
Here's my LogOutput if that helps.

[LogOutput.log](https://github.com/Okabintaro/SubmersedVR/files/12037210/LogOutput.log)

**Reply by Okabintaro** [reply]
You are using a very old version of SubmersedVR(0.1.2). Please use the latest one.

**Reply by tumja001** [reply]
[LogOutput.log](https://github.com/Okabintaro/SubmersedVR/files/12040727/LogOutput.log)

Cheers. I uninstalled Subnautica, deleted the folder and started from scratch, ensuring I had the latest files.
No change. Here's the log.

**Reply by Okabintaro** [reply]
You have to copy the whole contents of the mod folder, not just the .dll to the plugins, like it is mentioned in the README.
See https://github.com/Okabintaro/SubmersedVR/discussions/50#discussioncomment-6332469

**Comment by tumja001** [comment]
Cheers, I had just extracted the zip into the main folder when manually installing, but hadn't done that with the latest version when using Vortex. Is it the Subnautica_Data folder that I'd missed?
That's working a treat now. Looks great.

**Reply by Okabintaro** [reply]
Yes, please just do the manual installation. You don't need vortex for this. And installing it manually shouldn't break vortex either.

**Reply by tumja001** [reply]
No problem. I'm not worried about Vortex anyway, I only used it because of the instructions for installing the latest version of Tobey's BepInEx Pack.

**Reply by Okabintaro** [reply]
I see, that one you can find on github too though: https://github.com/toebeann/BepInEx.Subnautica/releases/tag/v5.4.21-payload.1.5.1

**Comment by natoea** [comment]
hi im having same issue with submersedvr cant get the thing to work i followed the steps to the letter but just not working any advice would be very helpful
